packages/sdk/src/fetcher/subgraph/index.ts

```diff
@@ -198,15 +202,57 @@ class Fetcher implements IPartialCarrotFetcher {
             : tokens.map(({ rawAddress }) => getAddress(rawAddress));
     }
 
+    public normalizeKpiTokens = async (
+        rawTokensList: KPITokenData[] | KPITokenSearchData[],
+        chainId: ChainId,
+        ipfsGatewayURL: string
+    ) => {
+        const kpiTokens:
+            | Promise<{ [address: string]: KPIToken }>
```

Comment:
Pretty sure we could just use `{ [address: string]: KPIToken }` without the promise variant? TS should understand that.

Reply:
I'll check.

Reply:
Yes. Removed. thanks.